### DISCUSSION TITLE
clickhouse query not working

### DISCUSSION THREAD

**Discussion by hardik-codept** [plausible/analytics#4567] [Self-Hosted Support]
my requirement is that,

I have a page for ex domain.com/contact-us I am changed the the page slug to domain.com/contact

In state domain.com/contact consider as new page.

I want move old path data to new path so If I checked page state it would be same. and delete the old path data.

I have try this commands. 

```
:) alter table events_v2 update pathname = 'contact' where pathname = 'contact-us';
:) alter table sessions_v2 update entry_page = 'contact' where entry_page = 'contact-us';
:) alter table sessions_v2 update exit_page = 'contact' where exit_page = 'contact-us';
```

in my case I want change path `hofejt` to `konetas` 

It's show `0 rows in set.`  - https://prnt.sc/9IzlHC8WW_2m

But you can see when I run select query we can get result where path is `hofejt`
- https://prnt.sc/jkv-az-J58DE

Can you please assist me for this issue? 

Thank you.

**Comment by ruslandoga** [comment]
👋 @hardik-codept 

Your query updates konetas page to hofejt, you seem to want the opposite.
